libraries/AP_InertialSensor/AP_InertialSensor_Backend.cpp

```diff
@@ -388,9 +388,26 @@ void AP_InertialSensor_Backend::_notify_new_accel_raw_sample(uint8_t instance,
         _imu._delta_velocity_acc[instance] += accel * dt;
         _imu._delta_velocity_acc_dt[instance] += dt;
 
-        _imu._accel_filtered[instance] = _imu._accel_filter[instance].apply(accel);
-        if (_imu._accel_filtered[instance].is_nan() || _imu._accel_filtered[instance].is_inf()) {
+        // apply the low pass filter
```

Comment:
You will need to change this logic to match https://github.com/ArduPilot/ardupilot/pull/14470